frontend/widgets/OBSBasic.hpp

```diff
@@ -1470,6 +1470,14 @@ private slots:
 	std::vector<OBSDataAutoRelease> safeModeTransitions;
 	QPointer<QPushButton> transitionButton;
 	QPointer<QMenu> perSceneTransitionMenu;
+
+	std::unordered_map<std::string, OBSSource> transitions;
+	/* NOTE: There is a reliance on the order of insertion */
```

Comment:
Use C++ comment style, though I think this comment might be unnecessary as a `std::vector` always keeps the insertion order and code usually relies on that anyway.

If we want to keep the note it should be rephrased into something like:

```suggestion
	// NOTE: Any code accessing this collection relies on order of insertion
```

Reply:
This note tries to indicate that we should go towards dropping this reliance on order.